docs/persistence/postgres/how-to/migrate-to-new-instance.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Migrate to new instance
+tags: [postgres, migrate, how-to]
+---
+
+# Migrating to a new SQLInstance
+
+This guide describes how to migrate your PostgreSQL database to a new [SQLInstance](../explanations/cloud-sql-instance.md).
+
+## Preparations
+
+* During migration, you can't make DDL changes (changing structure of tables, adding or removing tables, etc.) to your database. 
+  Check with your team to avoid any problems during the migration.
+* Make sure to have the latest version of the [nais CLI](../../../operate/cli/README.md) installed.
+* Decide on a new name for the new SQLInstance. It must be different from the current SQLInstance name.
+* naisdevice connected.
+* Access to the nais cluster where the application is running.
+
+## Setting up the migration
+
+1. Select the correct kubernetes context and namespace
+
+   ```shell
+   kubectl config use-context <cluster>
+   kubectl config set-context --current --namespace=<namespace>
+   ```
+
+2. Run the [setup](../../../operate/cli/reference/postgres.md#migrate-setup) command and follow the prompts
+    
+    ```shell
+    nais migrate postgres setup <appname> <new-sql-instance-name>
+    ```
+
+3. Check that the new instance contains expected data
```

Comment:
How do we check that the new instance contains expected data?
What data are we expecting?

Reply:
Maybe
```suggestion
3. Check that the new instance contains the data you expect (AKA same as old instance)
```

Reply:
Ga opp å forklare hvordan man gjør dette, så tok det vekk :sweat_smile: